internal/guest/storage/overlay/overlay.go

```diff
@@ -61,9 +61,8 @@ func processErrNoSpace(ctx context.Context, path string, err error) {
 func MountLayer(
 	ctx context.Context,
 	layerPaths []string,
-	upperdirPath, workdirPath, rootfsPath string,
+	rootfsPath, upperdirPath, workdirPath string,
```

Comment:
why the reordering of parameters? i dont see how this is related to adding encrypted scratch. am i missing something or did you decide that it was better this way and decided to tack it on here? if the latter, i think it would be better as a different PR so people like me don't start going, "why was that done"?